omnia/config.sh

```diff
@@ -181,6 +183,20 @@ importOptionsEnv () {
 	[[ ${errors[*]} ]] && { printf '%s\n' "${errors[@]}"; exit 1; }
 }
 
+importServicesEnv () {
+	local _config="$1"
+	local _json
+
+	_json=$(jq -S '.services' < "$_config")
+	#extract cmc key
+	CMC_API_KEY="${CMC_API_KEY-$(jq -r '.cmcApiKey' <<<"$_json")}"
+	#validate cmc key
+	[[ "$CMC_API_KEY" =~ ^[0-9a-f]{8}(-){1}[0-9a-f]{4}(-){1}[0-9a-f]{4}(-){1}[0-9a-f]{4}(-){1}[0-9a-f]{12}$ ]] || errors+=("Error - Coinmarketcap API Key is invalid.")
```

Comment:
`(-){1}` can be simplified to just `-`.

```suggestion
	[[ "$CMC_API_KEY" =~ ^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$ ]] || errors+=("Error - Coinmarketcap API Key is invalid.")
```